index.js

```diff
@@ -99,16 +143,59 @@ function createRenderer() {
 }
 
 /**
+ * Get service iframe video by passed name
+ * Required href:
+ * - youtube: https://youtu.be/1GWoMnYldYc or https://www.youtube.com/watch?v=1GWoMnYldYc
+ * - yandex: https://video.yandex.ru/iframe/ya-events/clg24o8tu3.7046/
+ * - vimeo: https://vimeo.com/30838628 or https://player.vimeo.com/video/30838628
+ * @param {String} href - full video href, examples see above
+ * @param {String} serviceName - name of the service [youtube, yandex, vimeo]
+ * @returns {String} - iframe video block
+ */
+function getIframe(href, serviceName) {
+    var serviceToUrl = {
+            youtube: 'www.youtube.com/embed/',
+            yandex: 'video.yandex.ru/iframe/ya-events/',
+            vimeo: 'player.vimeo.com/video/'
+        },
+        arr = href.split('/'),
+        key = arr.pop(),
+        src;
+
+    /**
+     * Get the second last href arr item for key,
+     * if href contains slash at the end and our key empty
+     * For example: https://vimeo.com/53219242/
+     */
+    if (!key.length) {
```

Comment:
i think it's better to get the key in the method above where you test url for service. when you know service chunk it's much easier to get a `key`.